### DISCUSSION TITLE
url

### DISCUSSION THREAD

**Discussion by franckbertuzzi** [GetPublii/Publii#1851] [Help]
Hello to the community
I'd like to build pages with this url structure : 

For exemple : 
* With the word "speciality"
domain.com/speciality/name_page1
domain.com/speciality/name_page2
domain.com/speciality/name_page3

* With the world "communication"
domain.com/communication/name_page1
domain.com/communication/name_page2
domain.com/communication/name_page3

What is the best way ?
Thank you

**Comment by bobmitro** [comment]
Here’s how URLs work in Publii: [How URLs in Publii Work](https://getpublii.com/dev/how-urls-in-publii-works)

**Comment by franckbertuzzi** [comment]
Thank you Bob
I forgot to active "use pretty url's"...
All is good
Thank you !
